peer/ping_manager.go

```diff
@@ -18,7 +18,7 @@ type PingManagerConfig struct {
 	NewPingPayload func() []byte
 
 	// NewPongSize is a closure that returns a random value between
-	// [0, lnwire.MaxPongBytes]. This random value helps to more effectively
+	// [1, 4096]. This random value helps to more effectively
```

Comment:
nit: `4096` -> `PongSizeCeiling`

Reply:
Actually gotta change this to MaxPongBytes since the choice to use PongSizeCeiling is up to the caller. The PingManager module's only requirement is that the closure is bounded `[0, MaxPongBytes]`.